lib/src/st2110/st_fmt.c

```diff
@@ -57,6 +57,13 @@ static const struct st20_pgroup st20_pgroups[] = {
         .coverage = 4,
         .name = "ST20_FMT_YUV_420_12BIT",
     },
+    {
+        /* ST20_FMT_YUV_420_16BIT */
+        .fmt = ST20_FMT_YUV_420_16BIT,
```

Comment:
You added the missing enum value (ST20_FMT_YUV_420_16BIT = 7). Great!
I wonder (not due to your changes but nearby consistently), if the ST20_FMT_YUV_422_8BIT=1 and ST20_FMT_YUV_422_10BIT=0 (first 2 values in this array) should not be also changed between each other? (to make order in the values list? (Now it is: 1, 0, 2, 3, 4, ..., 17.))

Reply:
Done